File: test-project-javascript/src-javascript/language_adapters/infrastructure_raw_literal_adapter.ts
// PURPOSE: Test AES0304 — raw literals in infrastructure
export class RawLiteralAdapter {
    timeout: number = 30;
    host: string = "localhost";
    port: number = 8080;

    connect(): string {
        const url = "http://example.com";
        const retries = 3;
        return url;
    }
}

File: test-project-python/src-python/language_adapters/infrastructure_raw_literal_adapter.py
# PURPOSE: Test AES0304 — raw literals in infrastructure
class RawLiteralAdapter:
    def __init__(self):
        self.timeout = 30
        self.host = "localhost"
        self.port = 8080
        self.debug = True

    def connect(self):
        url = "http://example.com"
        retries = 3
        return url
